lib/linguist/generated.rb

```diff
@@ -96,6 +97,20 @@ def minified_files?
       end
     end
 
+    # Internal: Is the blob a generated source map?
+    #
+    # Source Maps usually have .css.map or .js.map extensions. In case they
+    # are not following the name convention, detect them based on the content.
+    #
+    # Returns true or false.
+    def source_map?
+      return false unless extname.downcase == '.map'
+
+      name =~ /(\.css|\.js)\.map$/i ||                 # Name convention
+      lines[0] =~ /^{"version":\d+,/ ||                # Revision 2 and later begin with the version number
```

Comment:
@arfon That is to avoid calling `.include?` on `nil`. I think no need for here like this one:
https://github.com/github/linguist/blob/master/lib/linguist/generated.rb#L106